cicecore/drivers/nuopc/cmeps/ice_import_export.F90

```diff
@@ -800,19 +834,10 @@ subroutine ice_import( importState, rc )
 
     if (.not.prescribed_ice) then
        call t_startf ('cice_imp_t2u')
-       call ice_HaloUpdate(uocn, halo_info, field_loc_center, field_type_scalar)
-       call ice_HaloUpdate(vocn, halo_info, field_loc_center, field_type_scalar)
-       call ice_HaloUpdate(ss_tltx, halo_info, field_loc_center, field_type_scalar)
-       call ice_HaloUpdate(ss_tlty, halo_info, field_loc_center, field_type_scalar)
-       ! tcraig, moved to dynamics for consistency
-       !work = uocn
-       !call grid_average_X2Y('F',work,'T',uocn,'U')
-       !work = vocn
-       !call grid_average_X2Y('F',work,'T',vocn,'U')
-       !work = ss_tltx
-       !call grid_average_X2Y('F',work,'T',ss_tltx,'U')
-       !work = ss_tlty
-       !call grid_average_X2Y('F',work,'T',ss_tlty,'U')
+       call t2ugrid_vector(uocn)
```

Comment:
I'm getting a compile error for t2ugrid_vector, it has been removed from ``cicecore/cicedynB/infrastructure/ice_grid.F90`` and replaced by the ``grid_averageX2Y`` interface it seems. 

I'm not sure I understand the tcraig comment; maybe this entire block should be removed.

Reply:
uocn, vocn, ss_tltx, ss_tlty are now assumed to be on the U grid and the averaging to the T grid is being done in CICE.  It no longer needs to be done here.   We are moving to a more general system in CICE where the location of the coupling/forcing variables are more clearly defined within CICE.  This should be carefully reviewed in the context of CESM coupling to make sure it's working properly.